spatial/curve/curve.go

```diff
@@ -0,0 +1,23 @@
+// Package curve defines space filling curves. A space filling curve is a curve
```

Comment:
Each file will need to have a copyright header with 2021 as the copyright data. You can just copy/paste from another file in the repo.